docs/next/modules/en/pages/user/clusters.adoc

```diff
@@ -10,7 +10,7 @@ Keep in mind that most Cluster API Providers are upstream projects maintained by
 
 [tabs]
 ======
-AWS EKS/RKE2/Kubeadm::
+AWS EKS::
```

Comment:
Should this just say `AWS`? Because below we cover both EKS and "bring your own AWS infra" provider options.